runtime/astar/src/lib.rs

```diff
@@ -1207,7 +1235,14 @@ pub type Executive = frame_executive::Executive<
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
-pub type Migrations = (cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,);
+pub type Migrations = (
+    cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,
+    // permanent migration, do not remove
+    pallet_xcm::migration::MigrateToLatestXcmVersion<Runtime>,
```

Comment:
I don't understand

Reply:
The comment implies we should permanently keep this migration here.
I'm just wondering if we should really do this - removing this line AFTER the migration is finished seems more prudent to me.

Thanks to the CI checks you added, we can easily detect if some migration is missing.

Reply:
@ashutoshvarma ^

Reply:
We can do both, but I preferred permanent so that we don't forgot when new xcm is released.
I'm not sure how we can detect this in CI, as this won't break anything even if this is missed.

Reply:
`try-runtime` checks will detect that on-chain version differs from the version defined in the pallet code and it will throw an error. So we cannot miss any migrations anymore.

Reply:
Ok, but a new xcm version will not mean a new pallet version for `pallet_xcm`, those are different.